hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -193,6 +198,67 @@ private void init() throws Exception {
 
     // stop the deletion services so that keys can still be read
     keyManager.stop();
+    preFinalizationChecks();
+    finalizeOMUpgrade();
+  }
+
+  private static void expectFailurePreFinalization
+      (LambdaTestUtils.VoidCallable eval)
+      throws Exception {
+    LambdaTestUtils.intercept(OMException.class,
```

Comment:
You can directly use `Assert.assertThrows` instread of `LambdaTestUtils`.

Reply:
Yup! that makes sense!